tests/examples/opinions/united_states/colo_example_3.compare.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "case_dates": "2022-02-14",
+    "case_names": "People v. Ray Ojeda",
+    "download_urls": "/userfiles/file/Court_Probation/Supreme_Court/Opinions/2019/19SC763.pdf",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "19SC763",
+    "citations": "22CO7",
```

Comment:
it occurs to me that this is not a proper citation format.  We should update the regex to miss this I think.  Or be smart enough to fix the citation.

Reply:
For this particular example, the citation is wrong, but that is how the court' staff listed this case in the Supreme Court web site.

In the original opinion file, the corrected citation is "**2022 CO 7**" and the opinion information is corrected with the scraped information from method **`extract_from_text`**; I'm not too worried about this cases like this. Should another validation be added for citations that don't follow the right format?

https://www.courts.state.co.us/userfiles/file/Court_Probation/Supreme_Court/Opinions/2019/19SC763.pd
![Screenshot from 2022-02-16 09-22-05](https://user-images.githubusercontent.com/8507838/154360702-01c0fcb5-20b4-46bd-a3ed-42a20e433664.png)

Reply:
yes - but we shouldn't be returning something that CL can't interpret as a citation.  Citations require spaces.

Reply:
I updated reporters-db to handle these permutations.

Reply:
Added a verification to discard bad formatted citations and dockets and notes explaining this change.